src/main/java/sirius/kernel/async/CallContext.java

```diff
@@ -359,6 +360,8 @@ public void removeFromMDC(String key) {
      *
      * @return a two-letter language code used for the current thread.
      */
+    @SuppressWarnings("deprecation")
+    @Explain("The replacement for NLS.getDefaultLanguage() is only available in sirius-web.")
```

Comment:
I'd say this should remain as sirius-web will provide a lazyLanguageInstaller (?)